Kevin-Hernandez.txt

```diff
@@ -0,0 +1,85 @@
+https://codepen.io/KevinH1/full/pogZRLv
```

Comment:
Some ideas to make your project look more like [the reference](https://codepen.io/BritHemming/full/jONmxOm)
- Look into the `box-sizing` property to make border "compress" the images instead of going on the outside
- Section 2 and 3 are flush against the body, instead of having whitespace around them. You put whitespace around the first section, try replicating this for 2 and 3.